### DISCUSSION TITLE
My coordinator is going crazy!

### DISCUSSION THREAD

**Discussion by salopette** [Koenkk/zigbee2mqtt#8551] [General]
My coordinator is going crazy!

Hello, is it normal for the coordinator to fidget like crazy on the map?

here is a video.

https://www.dropbox.com/s/ufrt3mkwo9i36v5/20210827_083434_1.mp4?dl=0

**Comment by Koenkk** [comment]
Frontend issues should be created here: https://github.com/nurikk/z2m-frontend/issues
